src/main/java/it/gov/pagopa/reward/notification/repository/RewardOrganizationExportsRepository.java

```diff
@@ -15,6 +16,7 @@ public interface RewardOrganizationExportsRepository extends ReactiveMongoReposi
 
     Flux<RewardOrganizationExport> findByStatusIn(Collection<RewardOrganizationExportStatus> statuses);
     Flux<RewardOrganizationExport> findByExportDate(LocalDate exportDate);
+    Mono<RewardOrganizationExport> findByOrganizationIdAndInitiativeIdAndId(String organizationId, String initiativeId, String exportId);
```

Comment:
let's move more restrictive condition as first
```suggestion
    Mono<RewardOrganizationExport> findByIdAndOrganizationIdAndInitiativeId(String exportId, String organizationId, String initiativeId);
```

Reply:
fix invocations to respect new order